content/_includes/page/list.njk

```diff
@@ -9,6 +9,11 @@
     collections
   ) }}
 {% elif bird and (bird != 'oddbird') %}
+  {{ confs.list(
+    title='Upcoming Events',
+    events=collections.all | byBird(bird) | getData('data.events'),
+    all=false
+  ) }}
```

Comment:
@mirisuzanne This adds an "Upcoming Events" title to all Bird pages, even if there are no events:

![Screenshot 2020-07-23 15 06 42](https://user-images.githubusercontent.com/552316/88328511-8befef80-ccf6-11ea-93b6-95bb39618efe.png)